packages/react-components/react-datepicker/src/components/Calendar/Calendar.types.ts

```diff
@@ -272,15 +261,15 @@ export interface CalendarStyles {
   /**
    * Style for the root element.
    */
-  root: IStyle;
+  root: string;
 
-  divider: IStyle;
+  divider: string;
 
-  goTodayButton: IStyle;
+  goTodayButton: string;
 
-  monthPickerWrapper: IStyle;
+  monthPickerWrapper: string;
 
-  liveRegion: IStyle;
+  liveRegion: string;
```

Comment:
For components that aren't using slots, `<component name>Styles` can take over the slots so we can have stronger types.